.github/ISSUE_TEMPLATE.md

```diff
@@ -1,4 +1,4 @@
 Please contribute feedback on the overall design of WebAssembly, issues about
 the textual specification, or typos from the docs section of webassembly.org.
 
-For browser issues or tooling bugs, see links at http://webassembly.org/community/feedback/.
\ No newline at end of file
+For browser issues or tooling bugs, see links at https://github.com/WebAssembly/website/blob/master/community/feedback.md.
```

Comment:
@jfbastien Re the Chair, my humble apologies I honestly thought you were claiming to be a Chair, but I see @titzer now listed as a Chair.